source/nodejs/adaptivecards-designer/src/open-sample-dialog.ts

```diff
@@ -9,9 +9,20 @@ class CatalogueItem {
 
     constructor(readonly entry: CatalogueEntry) { }
 
+    private static _id = 0;
+    private static _getNewItemId(prefix: string): string {
+        let newId = prefix + "-" + CatalogueItem._id;
+        CatalogueItem._id++;
+        return newId;
+    }
+
     render(): HTMLElement {
+        const newItemId = CatalogueItem._getNewItemId("acd-open-sample-item-title");
         let element = document.createElement("div");
```

Comment:
Style consistency: empty line after let declarations unless next line sets a property or calls a method on the newly declared variable